tools/StaticAnalysis/Exceptions/Az.App/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,19 @@
+"Module","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
why contain breaking change suppression in the breaking change notification PR?

Reply:
Whether the Location parameter is required or not does not affect already existing script files.
The parameter Location is now non-required, which is the correct code generated by correct parsing of swagger.


Regarding PassThru, it was confirmed through communication with xiaogang that this parameter should not be present in Get/New-Az*, and it will be fixed as a bug.
Meanwhile, I've confirmed with Alex that in the last 30 days, no one has called PassThru when using Get/New-Az* command, so we can just remove it.